selfdrive/ui/qt/onroad.cc

```diff
@@ -355,7 +355,7 @@ void AnnotatedCameraWidget::drawHud(QPainter &p) {
   int top_radius = 32;
   int bottom_radius = has_eu_speed_limit ? 100 : 32;
 
-  QRect set_speed_rect(QPoint(60 + (default_size.width() - set_speed_size.width()) / 2, 45), set_speed_size);
+  QRect set_speed_rect(QPoint((is_metric ? 60 : 45) + (default_size.width() - set_speed_size.width()) / 2, 45), set_speed_size);
```

Comment:
We can also match the margins of the other corner elements (experimental, dm, maps) but not sure of they should since they're different shapes.
```suggestion
  QRect set_speed_rect(QPoint((is_metric ? 45 : 30) + (default_size.width() - set_speed_size.width()) / 2, 30), set_speed_size);
```
<img width="1192" alt="Screenshot 2023-07-31 at 11 11 17 PM" src="https://github.com/commaai/openpilot/assets/1976269/6220bb98-a142-4833-820e-c991ea1192ad">